.github/workflows/cargo-update.yml

```diff
@@ -21,7 +25,8 @@ jobs:
         uses: peter-evans/create-pull-request@v4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          commit-message: "build: Bump dependencies"
+          commit-message: "build: Bump Cargo.lock dependencies"
+          title: "build: Bump Cargo.lock dependencies"
```

Comment:
I don't think the `build: ` prefix provides any value here. But I guess if this is going to become the standard format, it would be more surprising to see PRs without prefix.

Reply:
Yes...and there is a _possibility_ that the [conventional commit](https://www.conventionalcommits.org) style will be _enforced_ via PR titles...using something like the [Semantic Pull Requests GitHub App](https://github.com/zeke/semantic-pull-requests).

Reply:
Enforcing it on the PRs seems like the most logical place to enforce it. With squash as our default strategy, the commit messages on `main` are based on PR titles rather than actual commit messages (although the PR titles are often based on commit messages)